DigitalLearningSolutions.Web.Tests/ViewModels/TrackingSystem/Delegates/DelegateGroups/DelegateGroupsViewModelTests.cs

```diff
@@ -110,6 +113,7 @@ public void DelegateGroupsViewModel_filters_should_be_set()
             // Then
             model.Filters.Should().BeEquivalentTo(expectedFilters);
             model.DelegateGroups.First().Name.Should().BeEquivalentTo("K");
+            model.SearchString.Should().Be("K");

```

Comment:
Probably need to rename this test if we're asserting this as well now (or make a separate test for it)

Reply:
Yeah no worries. Done.